docs/topic_guides/evaluation/edge_metrics.md

```diff
@@ -0,0 +1,198 @@
+This guide is intended to be a reference guide for Edge Metrics used throughout Splink. It will build up from basic principles into more complex metrics.
+
+!!! note
+    All of these metrics are dependent on having a "ground truth" to compare against. This is generally provided by **Clerical Labelling** (i.e. labels created by a human). For more on how to generate this ground truth (and the impact that can have on Edge Metrics), check out the [Clerical Labelling Topic Guide](./labelling.md).
```

Comment:
Do we want to add a `(coming soon)` note alongside the `Clerical Labelling Topic Guide` link? It's not essential, but it feels strange that we are linking through to an unfinished page.

Reply:
Yea we can do - I guess my concern was I would forget to fix the (Coming soon) in this guide when the other one was done

Reply:
Haha, yeah, I figured that was probably your reservation. I'm happy to leave it as is for now.